webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx

```diff
@@ -117,11 +115,9 @@ export function TeamProfile({team, isArchived, onToggleArchive, isDisabled, save
                             onClick={noop}
                         >
                             {isArchived ? (
-                                <UnarchiveIcon
-                                    className='channel-icon channel-icon__unarchive'
-                                />
+                                <i className='icon icon-archive-arrow-up-outline'/>
```

Comment:
Should we be moving other icons back to these CSS classes instead of using the icon widgets? I would've figured the components were preferable, although I know we have many different ways of doing these anyway

Reply:
Yes, we should. Updating the icons wasn't the scope for this PR, but since I saw some files I was working on, and the icons looking weird, I just went ahead and updated it.

Reply:
Okay, that'd be good to know. We might want to chat about how to make those as easy as possible to use for other devs since we could even go through and replace things like the `UnarchiveIcon` component with a new component that just renders `<i className='icon icon-archive-arrow-up-outline'/>` directly as well